types/src/space.d.ts

```diff
@@ -72,12 +84,12 @@ export default class ColorSpace {
 	id: string;
 	aliases?: string[] | undefined;
 	base: ColorSpace | null;
-	coords: Coords;
+	coords: Record<string, CoordMeta>;
 	fromBase?: ((coords: Coords) => number[]) | undefined;
 	toBase?: ((coords: Coords) => number[]) | undefined;
 	formats: Record<string, Format>;
 	referred?: string | undefined;
-	white: string;
+	white: White;
```

Comment:
After looking at the code again I think this should be `White | string`

Definition of `this.white` in `space.js`: <https://github.com/LeaVerou/color.js/blob/main/src/space.js#L30>
`getWhite` function: <https://github.com/LeaVerou/color.js/blob/main/src/adapt.js#L10-L16>

```suggestion
	white: White | string;
```

Reply:
Oh yeah, you're right. I forgot it was `white` for the instance property and not for the options object